Libplanet/Store/BaseStore.cs

```diff
@@ -23,23 +23,21 @@ public abstract class BaseStore : IStore, IDisposable
 
         public abstract long CountIndex(Guid chainId);
 
-        public abstract IEnumerable<HashDigest<SHA256>> IterateIndexes(
-            Guid chainId,
-            int offset,
-            int? limit);
+        /// <inheritdoc cref="IStore.IterateIndexes(Guid, int, int?)"/>
+        public abstract IEnumerable<BlockHash> IterateIndexes(Guid chainId, int offset, int? limit);
 
-        public abstract HashDigest<SHA256>? IndexBlockHash(Guid chainId, long index);
+        /// <inheritdoc cref="IStore.IndexBlockHash(Guid, long)"/>
+        public abstract BlockHash? IndexBlockHash(Guid chainId, long index);
 
-        public abstract long AppendIndex(
-            Guid chainId,
-            HashDigest<SHA256> hash
-        );
+        /// <inheritdoc cref="IStore.AppendIndex(Guid, BlockHash)"/>
+        public abstract long AppendIndex(Guid chainId, BlockHash hash);
 
-        /// <inheritdoc />
+        /// <inheritdoc cref="IStore.ForkBlockIndexes(Guid, Guid, BlockHash)"/>
```

Comment:
is `cref` in `inheritdoc` needed?

Reply:
It's not mandatory in theory, but since I've observed [DocFX] did not render `<inheritdoc>`s without `cref`, I tend to give them explicit `cref`.

[DocFX]: https://dotnet.github.io/docfx/